Resources/Prototypes/_DEN/Species/Ovinia.yml

```
@@ -0,0 +1,154 @@
+- type: species
+  id: Ovinia
+  name: species-name-ovinia
+  roundStart: true
+  prototype: MobOvinia
+  sprites: MobOviniaSprites
+  defaultSkinTone: "#ffffff"
+  markingLimits: MobOviniaMarkingLimits
+  dollPrototype: MobOviniaDummy
+  skinColoration: Hues
+  maleFirstNames: NamesFirstMale
+  femaleFirstNames: NamesFirstFemale
+  lastNames: NamesLast
+  minHeight: 0.5
```

default values, remove
```suggestion
  skinColoration: Hues
  minHeight: 0.5
```

This minHeight makes them BY FAR the smallest species in dV. Avali have 0.8, is this intended?

---

Fixed, updated to basescale to match other species YML file styles